src/components/templates/ClubCard/index.tsx

```diff
@@ -135,7 +138,7 @@ const ClubCard: FC<Props & RouteComponentProps> = ({ id, name, description, imag
             <CardDescription>
                 <ClubNameText>{name}</ClubNameText>
                 <ClubInfo>
-                    <HeartImg src={likeEmptySvg} /> {likes}
+                    <HeartImg src={liked==true? likeFilledSvg : likeEmptySvg} /> {likes}
```

Comment:
```suggestion
                    <HeartImg src={liked ? likeFilledSvg : likeEmptySvg} />{likes}
```

true 지우고 띄어쓰기 맞춰주면 좋을 것 같아여

Reply:
여기서 바로 Commit  Suggestion 누르면 되나보다! 😮